.circleci/config.yml

```diff
@@ -18,6 +18,9 @@ defaults:
     FLOWAPI_FLOWDB_USER: flowapi
     FLOWMACHINE_FLOWDB_PASSWORD: foo
     FLOWAPI_FLOWDB_PASSWORD: foo
+    ALEMBIC_FLOWDB_USER: flowdb
```

Comment:
These'll need to be passed into the FlowDB spinups that happen in the flowdb build job I think.